### PR TITLE
fix(navtabs): refer a friend tab selection no longer select discover tab

### PR DESCRIPTION
**Pull request by ak-ledger**
### 📝 Description

Discover tab can no longer be selected at the same time as the refer a friend one.

### ❓ Context

- **Impacted projects**: `LLD` 
- **Linked resource(s)**: [LIVE-8869] 

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 📸 Demo

Screenshot 2023-10-25 at 12 17 32

[LIVE-8869]: https://ledgerhq.atlassian.net/browse/LIVE-8869?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by cgrellard-ledger** [comment]
No need for the useMemo here ;)

**Comment by KVNLS** [comment]
Can you create a documentation, please? https://ledgerhq.atlassian.net/browse/LIVE-8869?focusedCommentId=369380

File: apps/ledger-live-desktop/src/renderer/components/MainSideBar/index.tsx
[the remark this answers is not in the document]
Comment (mcayuelas-ledger) [reply apps/ledger-live-desktop/src/renderer/components/MainSideBar/index.tsx:1]: cc: @KVNLS @cgrellard-ledger

**Review by KVNLS — APPROVED** [review]
LGTM, just a typescript error to fix.

**Review by mcayuelas-ledger — APPROVED** [review]
LGTM

**Pull request merged**